Modify the effort appendix in DG (#214)
AY2122S1-CS2103-F10-1/tp: docs/DeveloperGuide.md

## **Appendix: Effort**
**<u>Difficulty: Medium to Hard</u>**

Our project requires a fair amount of understanding of the code base. Although some commands implemented were based off of AB3, we introduced the interaction between `Position` and `Applicant`.

> The rule that the position each applicant applies to must exist in MrTechRecruiter cannot be violated after any commands, and it is not easy to ensure that.

Reviewer: ```suggestion
The rule that the position each applicant applies to must exist in MrTechRecruiter cannot be violated after any commands, which is difficult to ensure.
```

> In addition to the great enhancement based on original features, some commands were freshly added in to be in tune with our product, which requires understanding of JavaFX and careful design. 

Reviewer: ```suggestion
In addition to significant enhancements based on original features, some commands were freshly added in to be in tune with our product, which requires understanding of JavaFX and careful design. 
```
